cmd/release/cmd/update.go

```diff
@@ -253,6 +252,16 @@ var updateCLICmd = &cobra.Command{
 	},
 }
 
+func configRancherVersions() []string {
```

Comment:
Can we use `copy` here instead?

Reply:
Ah, I get it now. The reason why I didn't use it is that `Rancher.Versions` is a `map[string]` and not a slice